README.md

```diff
@@ -122,6 +122,70 @@ If you are using the gui, we recommend that you activate advanced mode. Also you
 
 After generating the build configuration, just open the Sourcetrail.sln file that was generated by CMake and build the Sourcetrail project.
 
+##### If using clang-cl.exe and Ninja (such as via CLion)
```

Comment:
This is interesting information, but I'm not sure we want it here. @mlangkabel wanted to revise the README. What do you think?

Reply:
My preference would be to break the info in the readme into multiple files, perhaps re-organized by platform.

Reply:
Sounds good!

Reply:
@LouisStAmour, thanks for providing setup instructions for using Clang/Ninja on Windows. Currently the document was structured this way:

* target 1
  * dependencies
    * dependency 1
    * ...
    * dependency N (windows only)
    * dependency M (unix only)
  * build instructions
    * instruction windows
    * instruction unix
* target 2
  * ...
We chose this approach because it allows us to keep all (cross-platform) information for a target together in one spot. So if we update a dependency version we don't have 3 documents where we need to remember to make the update. And that's why I would like to keep it that way. 

However, the format of this whole "Building Sourcetrail" section is still horrible to read. I was working on some improvements. Maybe I can get that done today. This will probably cause conflicts here, but you would need to split the information provided here into the individual sections (e.g. dependencies, building).